packages/firebase/index.d.ts

```diff
@@ -5711,8 +5786,15 @@ declare namespace firebase.database {
      *
      * @param host the emulator host (ex: localhost)
      * @param port the emulator port (ex: 8080)
+     * @param options.mockUserToken - the mock auth token to use for unit testing Security Rules
```

Comment:
The jsdoc format read by the "old" docgen tool doesn't like dashes (`-`).
The dashes are required in the exp doc comments though. Sorry, confusing.